docs/rst/knora-api-server/api_v1/create-a-mapping.rst

```diff
@@ -0,0 +1,358 @@
+                <attributes>
+                    <attribute>
+                        <attributeName>description</attributeName>
+                        <namespace>noNamespace</namespace>
+                        <propertyIri>http://www.knora.org/ontology/anything#standoffEventTagHasDescription</propertyIri>
+                    </attribute>
+                </attributes>
+                <datatype>
+                    <type>http://www.knora.org/ontology/knora-base#StandoffDateTag</type>
+                    <attributeName>knoraDate</attributeName>
+                </datatype>
+            </standoffClass>
+        </mappingElement>
+    <mapping>
+
+``<datatype>`` **must** hold the Iri of a standoff data type class (see list above). The ``<classIri>`` must be a subclass of this type or this type itself (the latter is probably not recommendable since semantics are missing: what is the meaning of the date?).
+In the example above, the standoff class is ``anything:StandoffEventTag`` which has the following definition in the ontology `anything-onto.ttl``::
+
+
+    anything:StandoffEventTag rdf:type owl:Class ;
+
+        rdfs:subClassOf knora-base:StandoffDateTag,
+                       [
+                          rdf:type owl:Restriction ;
+                          owl:onProperty :standoffEventTagHasDescription ;
+                          owl:cardinality "1"^^xsd:nonNegativeInteger
+                       ] ;
+
+        rdfs:label "Represents an event in a TextValue"@en ;
+
+        rdfs:comment """Represents an event in a TextValue"""@en .
+
+
+``anything:StandoffEventTag`` is a subclass of ``knora-base:StandoffDateTag`` and therefore has the data type date.
+It also requires the standoff property ``anything:standoffEventTagHasDescription`` which is defined as an attribute in the mapping.
+
+Once the mapping has been created, an XML like the following could be sent to Knora and converted to standoff::
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <text>
+        We had a party on <mydate description="new year" knoraDate="GREGORIAN:2016-12-31">New Year's Eve</mydate>. It was a lot of fun.
+    </text>
+
+
+The attribute holds the date in the format of a Knora date string (the format is documented in the typescript type alias ``dateString`` in module ``basicMessageComponents``. There you will also find documentation about the other types like color etc.).
+
+You will find a sample mapping with all the data types and a sample XML file in the the test data: ``webapi/_test_data/test_route/texts/mappingForHTML.xml`` and ``webapi/_test_data/test_route/texts/HTML.xml``.
+
+-------------------
+Standoff Properties
+-------------------
+
+When mapping XML attributes to standoff properties, attention has to be paid to the properties' object constraints. Standoff properties are literals with the exception of internal references that are pointers to standoff nodes.
```

Comment:
> Should there be a class StandoffInternalReferenceTag

That's a good question. 
So far, the standoff data types reflect the set of existing value types and are exclusive: either a standoff class is of type date or integer or it has no type at all, but types cannot be combined.

Not making internal references a data type would allow - as you say - any `StandoffTag` to have that property.

If we decided to make internal reference a datatype, this would mainly require some work in `/Users/tobi/gitreps/Knora_github/webapi/src/main/scala/org/knora/webapi/util/standoff/StandoffTagUtilV1.scala`.